apps/src/applab/redux/screens.js

```diff
@@ -1,10 +1,29 @@
 import Immutable from 'immutable';
+import {
+  sources as sourcesApi,
+  channels as channelsApi,
+} from '../../clientApi';
 
 const CHANGE_SCREEN = 'screens/CHANGE_SCREEN';
 const TOGGLE_IMPORT_SCREEN = 'screens/TOGGLE_IMPORT_SCREEN';
+const IMPORT = {
+  IMPORT_PROJECT: {
+    START_FETCHING: 'screens/importProject/START_FETCHING',
+    FAILED_FETCHING: 'screens/importProject/FAILED_FETCHING',
+    FINISHED_FETCHING: 'screens/importProject/FINISHED_FETCHING',
+  },
+};
+
+var ImportProjectState = Immutable.Record({
```

Comment:
I dont feel super strongly, but I find these nested sets of all caps hard to read. I guess another option would be to just have something like `import.importProject.START_FETCHING`. Not sure if that's weirder. Again, dont feel super strongly.